docs/styling.md

```diff
@@ -80,7 +80,7 @@ Astro also supports [Sass][sass] out-of-the-box. To enable for each framework:
 
 ### 🦊 Autoprefixer
 
-We also automatically add browser prefixes using [Autoprefixer][autoprefixer]. By default, Astro loads the [Browserslist defaults][browserslist-defaults], but you may also specify your own by placing a [Browserslist][browserslist] file in your project root.
```

Comment:
`[browserslist-defaults]` was a reference to a missing link reference definition at the bottom of the file. See https://github.com/snowpackjs/astro/pull/359 for how to add a new link reference definition

Reply:
Oops! Thanks for the clarification, done!